elrond.md

```diff
@@ -521,6 +543,14 @@ TODO: Implement [reserved keys and read-only runtimes](https://github.com/Elrond
            ...
          </account>
 
+    rule <instrs> #storageLoadFromAddress => . ... </instrs>
+         <bytesStack> ADDR : KEY : STACK => #lookupStorage(STORAGE, KEY) : STACK </bytesStack>
```

Comment:
Rules that, for some inputs, become `lhs => #Bottom`, where lhs is not bottom, are invalid (their axiom becomes just #Bottom, which means that the semantics is inconsistent).

Here, `#lookupStorage` is a partial function, which means that the rhs can be bottom, so you need to add `requires whatever-is-needed-for-#lookupStorage-to-be-defined`.